Firestore/core/test/firebase/firestore/util/grpc_stream_tester.cc

```diff
@@ -82,7 +82,7 @@ std::string GetGrpcErrorCodeName(grpc::StatusCode error) {
   }
 }
 
-std::string GetFirestoreErrorCodeName(FirestoreErrorCode error) {
+std::string GetFirestoreErrorName(enum Error error) {
```

Comment:
It seems like this could be more generally useful defined as overloads of `util::ToString` and doesn't necessarily need to be isolated to just testing code. WDYT?

Feel free to punt to another PR.

Reply:
Sounds good, but yeah, I'd postpone it to another PR.